client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -555,7 +557,7 @@ export class PluginsList extends Component {
 					deactivateSelected={ this.deactivateSelected }
 					setAutoupdateSelected={ this.setAutoupdateSelected }
 					unsetAutoupdateSelected={ this.unsetAutoupdateSelected }
-					removePluginNotice={ this.removePluginDialog }
+					removePluginNotice={ () => this.removePluginDialog() }
```

Comment:
`removePluginDialog` now accepts `selectedPlugins` as an argument, which is not available here in this case. We are doing this to avoid the `event` being passed as an argument.